lib/generators/shopify_app/add_after_authenticate_job/add_after_authenticate_job_generator.rb

```diff
@@ -12,7 +12,9 @@ class AddAfterAuthenticateJobGenerator < Rails::Generators::Base
       def init_after_authenticate_config
         initializer = load_initializer
 
-       after_authenticate_job_config = "  config.after_authenticate_job = { job: Shopify::AfterAuthenticateJob, inline: false }\n"
```

Comment:
Is this just a line length linting change?

Reply:
Yup hehehe. Lint wasn't activated when this gem was last touched, for some reason. It started complaining about lots of things here. It was event using some general cops, we had to add Shopify's